### PR TITLE
graph-builder: add scopes for s390x

### PR DESCRIPTION
**Pull request by jlebon**
FCOS is now publishing s390x streams.

### CODE REVIEW

File: fcos-graph-builder/src/settings.rs [hunk 1]
```diff
@@ -34,10 +34,13 @@ impl ServiceSettings {
     /// Default TCP port for graph-builder main service.
     const DEFAULT_GB_SERVICE_PORT: u16 = 8080;
     /// Default scopes (basearch plus stream) to process.
-    const DEFAULT_SCOPES: [(&'static str, &'static str); 6] = [
+    const DEFAULT_SCOPES: [(&'static str, &'static str); 9] = [
         ("aarch64", "next"),
         ("aarch64", "stable"),
         ("aarch64", "testing"),
+        ("s390x", "next"),
+        ("s390x", "stable"),
```
Comment (lucab) [inline fcos-graph-builder/src/settings.rs:42]: I think this doesn't exist (yet?).
Were you planning to have this PR on hold until we have populated all the stream?

Comment (dustymabe) [reply fcos-graph-builder/src/settings.rs:42]: correct, stable will get created in two weeks. Does it hurt to have this configured before stable exists? i.e. should we wait and PR that separately in two weeks?

Comment (lucab) [reply fcos-graph-builder/src/settings.rs:42]: Yes, that would be cleaner, otherwise the scraper will be polling-then-erroring for all that time.

**Comment by LakshmiRavichandran1** [comment]
@lucab could you please approve and merge this?

**Pull request merged**